CHANGELOG.unreleased.md

```diff
@@ -1 +1,5 @@
 # Changelog Unreleased
+
+### Fixes 
+
+- [FRadio] Add an UID to radio to make the label clickable
```

Comment:
No entries for the fix on initial values on vee-validate version ? Maybe not relevant tho

Reply:
Oh yes you're right! Thanks